misc/protobuf-ballerina/src/main/resources/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
```suggestion
// Copyright (c) 2020, Google Inc.
```

Reply:
Do we really need to use these proto files provided by Google with their license?. I think we can use our own proto files with WSO2 license. 

On the other hand, if you have a real requirement of using these files, AFAIK you can use this with their license since it is also Apache 2.0. But it is better to provide the link that you copied from as a comment.

Reply:
These are some external files used. I included the source link as a comment.

Reply:
@daneshk, any idea of the licensing?